packages/checkbox-group/test/visual/lumo/checkbox-group.test.js

```diff
@@ -1,5 +1,6 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
+import '@vaadin/checkbox/theme/lumo/vaadin-checkbox.js';
```

Comment:
Same question as above, do we add this import just to be explicit?

Reply:
Fixed.